CHANGELOG.md

```diff
@@ -2,510 +2,321 @@
 
-  ([PR #1367](https://github.com/alphagov/govuk-frontend/pull/1367))
+Pull requests:
 
-- Simplify `.govuk-main-wrapper` logic to avoid the need for large modifier in most cases.
+- [#1434: Update font to use v2 of GOV.UK Transport font](https://github.com/alphagov/govuk-frontend/pull/1434)
+- [#1441: Removing font style adjustments](https://github.com/alphagov/govuk-frontend/pull/1441)
 
-  By using :first-child we can avoid the need for a modifier class, which is often missed.
+### Update links from error summary components to radios and checkboxes
 
-  We are also deprecating mixins for main wrapper.
-  We're not sure these are useful, so these will be removed in a future release, if you are using this please let us know: https://github.com/alphagov/govuk-frontend/issues/1379
+If you've linked from an error summary component to the first input in a [radios](https://design-system.service.gov.uk/components/radios/) or [checkboxes](https://design-system.service.gov.uk/components/checkboxes/) component, the link will no longer work.
 
-  ([PR #1371](https://github.com/alphagov/govuk-frontend/pull/1371))
+This is because the `id` of the first input no longer has the suffix `-1`.
 
-🔧 Fixes:
+If there are links back to radios or checkboxes components in your error summary component, remove `-1` from the end of the `href` attribute.
 
-- Removed adjustments that were needed for v1 Transport
+[Pull request #1426: Make radios and checkboxes components easier to link to from error summary](https://github.com/alphagov/govuk-frontend/pull/1426)
 
-  If v2 font is being used then there is no need for the adjustments that are currently used to compensate for the baseline issues of v1 Transport
+### Update start button icon
 
-  The following components were updated:
-  - Back-link
-  - Breadcrumbs
-  - Button (mostly affected start button)
-  - Header
-  - Tags
+[Start buttons](https://design-system.service.gov.uk/components/button/#start-buttons) have a new icon. Your start buttons will lose their current icons unless you replace the old icon with the new one.
 
-  If v1 font is being used then the adjustments will still automatically be included in the compiled css.
+If you're using Nunjucks:
 
-  ([PR #1441](https://github.com/alphagov/govuk-frontend/pull/1441))
+- set the `isStartButton` option to `true`
+- remove the `.govuk-button--start` class
 
-- Ensure character count message is hidden to assistive technologies when not visible
+For example:
 
-  ([PR #1442](https://github.com/alphagov/govuk-frontend/pull/1442))
+```javascript
+govukButton({
+  text: "Start now",
+  href: "#",
+  isStartButton: true
+})
+```
 
-- Fix issue where link underlines sit too low in Firefox
+If you're using HTML, add the SVG code from the [start button example in the Design System](https://design-system.service.gov.uk/components/button/#start-buttons).
```

Comment:
Maybe "HTML `<svg>` element"

Reply:
I've added this to our list of potential post-release style questions.